java/piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java

```diff
@@ -1564,6 +1565,39 @@ private boolean endsWithReturn(StatementTree stmt) {
     return false;
   }
 
+  /** removes enum in annotation and the comma if there is more than one enum in it */
+  private SuggestedFix.Builder deleteExprWithComma(
+      VisitorState state,
+      Tree tree,
+      ExpressionTree expressionTree,
+      ImmutableSet<ResolvedTestAnnotation> resolvedTestAnnotations,
+      SuggestedFix.Builder fixBuilder) {
+    for (ResolvedTestAnnotation resolvedTestAnnotation : resolvedTestAnnotations) {
+      if (resolvedTestAnnotation.getFlags().size() > 1) {
+        // the index is used to know if we have to remove the comma after or before the enum
+        int index =
+            IntStream.range(0, resolvedTestAnnotation.getFlags().size())
+                .filter(i -> resolvedTestAnnotation.getFlags().get(i).getValue().equals(xpFlagName))
+                .findFirst()
+                .orElse(-1);
+        if (index != -1) {
+          String replacementString = state.getSourceForNode(tree);
+          if (index == resolvedTestAnnotation.getFlags().size() - 1) {
+            replacementString =
+                replacementString.replace(", " + state.getSourceForNode(expressionTree), "");
```

Comment:
While I believe this will work in most cases, it still ties us down to a formatting style where arguments are separated by `, `. I believe we should make it more general, (e.g. arguments could also be separated by a new line character or arguments could be separated by just a `,`).
One options would be to do it the way error-prone folks delete type arguments in their [`UnnecessaryTypeArguments` bug check](https://sourcegraph.com/github.com/google/error-prone@69f45c3059b7b858dbe4c41afef6c20ae6ccaf15/-/blob/core/src/main/java/com/google/errorprone/bugpatterns/UnnecessaryTypeArgument.java?L83).